Jenkinsfile

```diff
@@ -80,6 +80,21 @@ def buildStaticAssets(env, tag) {
   stash name: "staticAssets${tag}", includes: "static${tag}.zip"
 }
 
+def cancelPreviousBuilds() {
+  def jobName = env.JOB_NAME
```

Comment:
jobName here is what, the branch name?

Reply:
` Name of the project of this build. This is the name you gave your job when you first set it up` So as this is a multibranch setups our build names follow the structure `pipeline_name/branch_name` e.g. `simorgh/cancel-previous-builds`